### PR TITLE
Run3-gex157A Replace use of cout's with LogVerbatim in Geometry/CaloTopology and Geometry/CommonTopologies

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Replace use of cout's with LogVerbatim in Geometry/CaloTopology and Geometry/CommonTopologies

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsnuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/CommonTopologies/test/ValidateRadial.cc [hunk 1]
```diff
@@ -40,7 +41,7 @@ ValidateRadial::ValidateRadial(const edm::ParameterSet& cfg)
       printOut_(cfg.getParameter<bool>("PrintOut")),
       posOnly_(cfg.getParameter<bool>("PosOnly")),
       tokTrackerGeometry_(esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>()) {
-  std::cout << "I'm ALIVE" << std::endl;
```
Comment (perrotta) [inline Geometry/CommonTopologies/test/ValidateRadial.cc:43]: The test script is looking for the string "ALIVE" in the output, see https://github.com/cms-sw/cmssw/blob/47f67fce7aaeaf012b9671049643ad4e66d6ba92/Geometry/CommonTopologies/test/runTest.sh#L5

If you move to LogVerbatim the script must be updated accordingly

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
+geometry

**Comment by perrotta** [comment]
+1

**Pull request merged**